src/js/rtp-stats.js

```diff
@@ -7,74 +7,102 @@
+                        jbMilliseconds:     when_defined(parseInt(statsReport.stat('googJitterReceived')))
                     };
 
                 } else if (is_defined(statsReport.packetsSent) && statsReport.mediaType == 'video' && streamType === 'video_input') {
                     extractedStats = {
                         timestamp:          timestamp,
-                        packetsCount:       statsReport.packetsSent,
-                        bytesSent:          statsReport.bytesSent,
-                        packetsLost:        is_defined(statsReport.packetsLost) ? Math.max(0, statsReport.packetsLost) : 0,
-                        rttMilliseconds:    when_defined(statsReport.googRtt),
-                        procMilliseconds:   is_defined(statsReport.googCurrentDelayMs),
-                        frameRateSent:      when_defined(statsReport.googFrameRateSent),
+                        packetsCount:       parseInt(statsReport.stat('packetsSent')),
+                        bytesSent:          parseInt(statsReport.stat('bytesSent')),
+                        packetsLost:        is_defined(statsReport.stat('packetsLost')) ? Math.max(0, statsReport.stat('packetsLost')) : 0,
+                        rttMilliseconds:    when_defined(parseInt(statsReport.stat('googRtt'))),
+                        procMilliseconds:   is_defined(parseInt(statsReport.stat('googCurrentDelayMs'))),
+                        frameRateSent:      when_defined(parseFloat(statsReport.stat('googFrameRateSent')))
                     };
 
                 } else if (typeof statsReport.packetsReceived !== 'undefined' && statsReport.mediaType == 'video' && streamType === 'video_output') {
                     extractedStats = {
                         timestamp:          timestamp,
-                        packetsCount:       statsReport.packetsSent,
-                        bytesReceived:      statsReport.bytesReceived,
-                        packetsLost:        is_defined(statsReport.packetsLost) ? Math.max(0, statsReport.packetsLost) : 0,
-                        frameRateReceived:  when_defined(statsReport.googFrameRateReceived),
-                        procMilliseconds:   is_defined(statsReport.googCurrentDelayMs),
-                        jbMilliseconds:     when_defined(statsReport.googJitterBufferMs)
+                        packetsCount:       parseInt(statsReport.stat('packetsSent')),
+                        bytesReceived:      parseInt(statsReport.stat('bytesReceived')),
+                        packetsLost:        is_defined(parseInt(statsReport.stat('packetsLost'))) ? Math.max(0, statsReport.stat('packetsLost')) : 0,
+                        frameRateReceived:  when_defined(parseFloat(statsReport.stat('statsReport.googFrameRateReceived'))),
+                        procMilliseconds:   is_defined(parseInt(statsReport.stat('googCurrentDelayMs'))),
+                        jbMilliseconds:     when_defined(parseInt(statsReport.stat('googJitterReceived')))
```

Comment:
statistics definition is changed, do we understand what is the new one exactly? and need to update stats document below

Reply:
Added new definition.